VULNERABILITY_RESPONSE_PROCESS.md

```diff
@@ -0,0 +1,147 @@
+# Monero Vulnerability Response Process
+
+## Preamble
+
+Researchers/Hackers: while you research/hack, we ask that you please refrain from committing the following:
+- Denial of Service / Active exploiting against the network
+- Social Engineering of Monero staff or contractors
+- Any physical or electronic attempts against Monero community property and/or data centers
+
+## I. Point of Contacts for Security Issues
+
+```
+ric@getmonero.org
+BDA6 BD70 42B7 21C4 67A9  759D 7455 C5E3 C0CD CEB9
+
+luigi1111@getmonero.org
+8777 AB8F 778E E894 87A2  F8E7 F4AC A018 3641 E010
+
+moneromooo.monero@gmail.com
+48B0 8161 FBDA DFE3 93AD  FC3E 686F 0745 4D6C EFC3
+
+jaquee.monero@gmail.com
+D21E 9CC1 2F51 C4FE A9E0  52FF 384E 52B0 9F45 DC39
```

Comment:
Linking would reduce redundancy, but I think we should let them either commit the key to the repo and/or decide where they want their fingerprint to reside.